### DISCUSSION TITLE
I can't see the Kibana Registry Dashboards. What gives?

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/registry#162] [Q&A] [answered]
When I go to the Kibana (aka OpenSearch) Dashboards interface, and try to go to the `Dashboard` page, but it doesn't look anything is loaded. What am I doing wrong?
 
<img width="1761" alt="Screenshot 2023-02-07 at 2 11 37 PM" src="https://user-images.githubusercontent.com/33492486/217378449-42936af3-fa2a-457b-99b7-fc84fbfe6574.png">

**Comment by pds-ops** [comment, marked as answer]
Make sure you check that you are logged in using the `Global Tenant`. The `Private Tenant` is great as a personal sandbox, but for Dashboards we want to share across nodes, we use the `Global Tenant`:

![image](https://user-images.githubusercontent.com/93455202/217379066-21040ba5-75fd-4aca-8a5d-8b7ab64f6eeb.png)

![image](https://user-images.githubusercontent.com/93455202/217379225-b9fdbd71-2b81-4204-8db7-b5a0de6dc394.png)

**Comment by jordanpadams** [comment]
Bah that was it! Thanks!
